### PR TITLE
First draft for discussion

### PR DESCRIPTION
**Pull request by josephinemorton**
@rrasins @kkousek @leecri @quietlibrarian5 Can you please check this for me?? - spelling is likely hopeless!!

### CODE REVIEW

**Comment by quietlibrarian5** [comment]
Looks fabulous

**Comment by leecri** [comment]
Great use of emoticons

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # josephinemorton.github.io
 
-This is my first website
+This is my first website :tada:
 
-This is my story of learning skills and tools that I can use in my job to make my life easier. It is all about *automating* tasks to more efficeintly use time, I don't want to do the **boring and mindless** tasks
+This is my story of learning skills and tools that I can use in my job to make my life easier. It is all about *automating* tasks to more efficeintly use time, I don't want to do the **boring and mindless** tasks.
```
Comment (rrasins) [inline README.md]: efficeintly should be efficiently

Comment (josephinemorton) [reply README.md]: Thanks @rrasins - knew there would be spelling mistakes

**Comment by rrasins** [comment]
Looks great! Just one spelling error :)

**Comment by josephinemorton** [comment]
@ccronje

**Comment by ccronje** [comment]
Excellent! Good to merge.

**Pull request merged**